### PR TITLE
Introducing torch.isinf

### PR DESCRIPTION
**Pull request by bhushan23**
Introduces torch.isinf for issue #9132

### CODE REVIEW

File: torch/functional.py [hunk 1]
```diff
@@ -322,6 +323,28 @@ def isnan(tensor):
...
+
+        >>> torch.isinf(torch.Tensor([1, float('inf'), 2]))
+         0
+         1
+         0
+        [torch.ByteTensor of size 3]
+    """
+    if not torch.is_tensor(tensor):
+        raise ValueError("The argument is not a tensor")
+    return (tensor == float('inf')) | (tensor == float('-inf'))
```
Comment (SsnL) [inline torch/functional.py]: this is three ops though... maybe this should be done as a custom elementwise op.

Comment (vishwakftw) [reply torch/functional.py]: One alternative would be to do: `tensor.abs() == float('inf')`

Comment (SsnL) [reply torch/functional.py]: yeah, we should at least do that. it's saving 1 op :)

Comment (bhushan23) [reply torch/functional.py]: Done! created new PR #9169

Comment (bhushan23) [reply torch/functional.py]: Closing this PR

**Pull request closed without merging**